color/d2v-color-common/src/main/kotlin/io/data2viz/color/RgbColor.kt

```diff
@@ -62,15 +52,13 @@ class RgbColor(override val rgb: Int, a: Double = 1.0): Color {
                 (rgb and 0xf).toString(16)
 
     override val rgba: String
-        get() = "rgba($r, $g, $b, $alpha)"
+        get() = "RGB($r, $g, $b, alpha=$alpha)"
```

Comment:
I missed that error. 

It's completely false. Ther rgab property is not here to do some king of debug information like `toString()`. It's used by JS renderer and respect [a specification](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#RGB_colors). You should check the usage of the property before doing this kind of modification.